docs/apache-airflow/tutorial.rst

```diff
@@ -154,7 +154,7 @@ references parameters like ``{{ ds }}``, calls a function as in
 ``{{ macros.ds_add(ds, 7)}}``, and references a user-defined parameter
 in ``{{ params.my_param }}``.
 
-The ``params`` hook in ``BaseOperator`` allows you to pass a dictionary of
+The ``params`` hook in ``BashOperator`` allows you to pass a dictionary of
```

Comment:
The original statement is correct.
params is a parameter of `BaseOperator`:
https://github.com/apache/airflow/blob/f2bd15da534c8a6ff802fe87992d1603737948e2/airflow/models/baseoperator.py#L475

`BashOperator` inherits from `BaseOperator`

Reply:
Not saying it isn't correct, but that while explaining the arguments of BashOperator, why suddenly refer to BaseOperator? It improves understanding if you refer to BashOperator instead which is also correct. The paragraph is explaining BashOperator rather than BaseOperator. 
Any special purpose in referring to BaseOperator instead of BashOperator there?

Reply:
I got confused suddenly reading BaseOperator there. It was not anywhere in the example code.